### PR TITLE
Add the option to choose what events to listen to

### PR DESCRIPTION
**Pull request by nadav5200**

### CODE REVIEW

**Comment by BoboTiG** [comment]
Thanks @nadav5200 !

Would you mind adding tests?

**Review by samschott — COMMENTED** [review]
Left a small comment about using inotify-specific event types in public API. I also assume this is still WIP since no actual filtering / selective listening is implemented yet?

File: src/watchdog/observers/api.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 from watchdog.utils import BaseThread
 from watchdog.utils.bricks import SkipRepeatsQueue
+from watchdog.observers.inotify_c import WATCHDOG_ALL_EVENTS
```
[the remark this answers is not in the document]
Comment (nadav5200) [reply src/watchdog/observers/api.py:24]: for sure!

**Comment by BoboTiG** [comment]
Thanks for the effort @nadav5200 :)
I merged #989 that achieve the same goal. Therefore I'm closing that PR.

**Pull request closed without merging**